pages/common/reflex.md

```diff
@@ -0,0 +1,24 @@
+# reflex
+
+> Reflex is a small tool to watch a directory and rerun a command when certain files change.
+> More information: <https://github.com/cespare/reflex>.
+
+- Rebuild with `make` if any file changes:
+
+`reflex make`
+
+- Compile and run Go application if any .go file changes:
```

Comment:
Perhaps this will read better?
```suggestion
- Compile and run a Go application if any `.go` files change:
```

Reply:
@sbrl great suggestion!

Reply:
@pjvds you should add the back ticks to highlight `.go` extension:
 ```suggestion
- Compile and run Go application if any `.go` file changes:
```